eng/scripts/Language-Settings.ps1

```diff
@@ -112,7 +114,7 @@ function Get-dotnet-PackageInfoFromPackageFile ($pkg, $workingDirectory)
 # Return list of nupkg artifacts
 function Get-dotnet-Package-Artifacts ($Location)
 {
-  $pkgs = Get-ChildItem "${Location}" -Recurse | Where-Object -FilterScript {$_.Name.EndsWith(".nupkg") -and -not $_.Name.EndsWith(".symbols.nupkg")}
+  $pkgs = @(Get-ChildItem $Location -Recurse | Where-Object -FilterScript {$_.Name.EndsWith(".nupkg") -and -not $_.Name.EndsWith(".symbols.nupkg")})
```

Comment:
The count doesn't work if it is not make the return array type literally. 
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1910727&view=logs&j=72a39880-3efd-52c2-15be-d04db16925c9&t=ea57d07a-07a4-5c12-9caa-17fa76bf1d24